server_installation/topics/clustering/booting.adoc

```diff
@@ -16,3 +16,6 @@ $ bin/domain.sh --host-config=host-master.xml
 $ bin/domain.sh --host-config=host-slave.xml
 ----
 
+It may be needed to use some additional parameters or system properties. For example parameter `-b` for the binding host. Or system property
+`jboss.node.name` to specify the name of the route as described in  <<sticky-sessions,Sticky Sessions >> section
+
```

Comment:
You may need to use additional parameters or system properties. For example, the parameter `-b` for the binding host or the system property `jboss.node.name` to specify the name of the route, as described in  <<sticky-sessions,Sticky Sessions >> section